src/main/java/ch/njol/skript/config/Node.java

```diff
@@ -125,11 +125,18 @@ public void move(final SectionNode newParent) {
 	 * leading #, except if there is no comment in which case it will be the empty string.
 	 * 
 	 * @param line
+	 * @param inBlockComment Whether we are currently inside a block comment
 	 * @return A pair (value, comment).
 	 */
-	public static NonNullPair<String, String> splitLine(final String line) {
-		if (line.trim().startsWith("#"))
+	public static NonNullPair<String, String> splitLine(String line, AtomicBoolean inBlockComment) {
+		String trimmed = line.trim();
+		if (trimmed.equals("###")) { // we start or terminate a BLOCK comment
+			inBlockComment.set(!inBlockComment.get());
+			return new NonNullPair<>("", trimmed);
```

Comment:
```suggestion
			return new NonNullPair<>("", line);
```
We should still pass the original line in case it is something like `###          `